offchain-modules/packages/app-rpc-server/src/server.ts

```diff
@@ -0,0 +1,94 @@
+import { rpcConfig, Config } from '@force-bridge/x/dist/config';
+import { ForceBridgeCore } from '@force-bridge/x/dist/core';
+import { startHandlers } from '@force-bridge/x/dist/handlers';
+import { getDBConnection } from '@force-bridge/x/dist/utils';
+import { logger, initLog } from '@force-bridge/x/dist/utils/logger';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import express from 'express';
+import { JSONRPCServer } from 'json-rpc-2.0';
+import { ForceBridgeAPIV1Handler } from './handler';
+import { GetBalancePayload, GetBridgeTransactionSummariesPayload, XChainNetWork } from './types/apiv1';
+
+const forceBridgePath = '/force-bridge/api/v1';
+const defaultLogFile = './log/force-bridge-rpc.log';
+
+export async function startRpcServer(config: Config) {
+  const rpcConfig: rpcConfig = config.rpc;
+  if (!config.common.log.logFile) {
+    config.common.log.logFile = defaultLogFile;
+  }
+  initLog(config.common.log);
+  config.common.role = 'watcher';
+
+  await new ForceBridgeCore().init(config);
+  const conn = await getDBConnection();
+  //start chain handlers
+  startHandlers(conn);
+  const forceBridgeRpc = new ForceBridgeAPIV1Handler(conn);
+
+  const server = new JSONRPCServer();
+  // First parameter is a method name.
+  // Second parameter is a method itself.
+  // A method takes JSON-RPC params and returns a result.
+  // It can also return a promise of the result.
+
+  /*
+      {
+        "jsonrpc": "2.0",
+        "method": "echo",
+        "params": { "text": "Hello, World!" },
+        "id": 1
+      }
+        =>
+      {
+        "jsonrpc": "2.0",
+        "id": 1,
+        "result": "Hello, World!"
+      }
+      * */
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  server.addMethod('echo', ({ text }) => text); //for test
```

Comment:
maybe change to the `info` with some meaningful information, such as `rpc-version`, `ckb-node-info`, etc.

Reply:
return rpc-version is a good idea